e2e/errors/cgm.errors.spec.js

```diff
@@ -0,0 +1,77 @@
+const {
```

Comment:
I'm guessing most of these changes are moving to a new file. As such, I'm just skimming.

Reply:
yep. that you are right about